lib/foreman_templates/engine.rb

```diff
@@ -28,6 +35,13 @@ class Engine < ::Rails::Engine
           }, :resource_type => 'Template'
         end
         add_all_permissions_to_default_roles
+
+        menu :top_menu, :template_sync,
+             :url_hash => { :controller => :template_syncs, :action => :index },
+             :caption => N_('Sync Templates'),
+             :parent => :hosts_menu,
+             :before => :ptables,
+             :turbolinks => false
```

Comment:
I'm not sure I like another link on the Navigation? Have you considered adding a link on both Provisioning, Partition and Job template pages instead? 

I don't really support either option more, just a discussion point

Reply:
That is a good idea, unfortunately the only way we can do it now is using Deface, which I would like to avoid. I'll gladly add them once we have proper extension points.

Reply:
Since as I understand where waiting for theforeman/foreman#6499 & theforeman/foreman#6604 is it worth adding a plugin API for it now? I'm not sure if that's a big task or not..